include/setup.h

```diff
@@ -184,30 +184,40 @@ class Property {
 	const Changeable::Value change;
 };
 
-class Prop_int:public Property {
+class Prop_int : public Property {
 public:
-	Prop_int(std::string const& _propname,Changeable::Value when, int _value)
-		:Property(_propname,when) {
-		default_value = value = _value;
-		min = max = -1;
+	Prop_int(const std::string &name, Changeable::Value when, int val)
+	        : Property(name, when),
+	          min_value(-1),
+	          max_value(-1)
```

Comment:
With std::min() and limits<..>::min and max burned in my mind, this change is much more readable. 
I'm actually surprised the compiler even allows names to be fully masked like this; or atleast didn't cry loud enough for the developers to care!

Reply:
Not sure, perhaps system headers that introduce min/max macros were not being included before `setup.h` anywhere (by sheer coincidence).